data/transport missions.txt

```diff
@@ -1975,3 +1975,46 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "Transfer enforcers to <planet>"
+	description "Transport 15 enforcers from <source> to <destination> by <day> to fulfill a request from the New Boston loan broker."
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
+		"unpaid mortgages" == 0
+		"combat rating" > 7
+		"net worth" > 5000000
```

Comment:
Someone with a net worth of 5 million credits is quite likely to have a combat rating higher than 7. :)

Reply:
Haha, you have any suggestions for the number? I don't know very much about what would be an accurate assumption so I'd rather ask someone than take another shot in the dark.

Reply:
I'd say at least 50.

Reply:
Just so we're clear, combat rating in the code is not the same as combat rating as displayed in-game; the latter is the natural logarithm of the former. So a rating of 7 is roughly the same as combat rating 2 in-game.